example-problems/gradle-project/build.gradle

```diff
@@ -13,21 +13,11 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
-buildscript {
-  repositories {
-    mavenCentral()
-    mavenLocal()
-    jcenter()
-  }
-  dependencies {
-    classpath 'com.google.cloud.tools:linkage-checker-gradle-plugin:0.1.0-SNAPSHOT'
-  }
```

Comment:
Because the plugin is available in Gradle plugin portal, it does not need to fetch the plugin from Maven local repo any more.